packages/arb-avm-cpp/fuzz_target/utils.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <avm/machine.hpp>
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+template <class... Args>
+void fuzz_require(bool check, Args... message) {
```

Comment:
I doubt performance is an issue here, but you likely want `Args&&...` for perfect forwarding. Also, nice variadic expansion below 😁

Reply:
Haha thanks I stole it from stack overflow. I'll fix it.